src/main/java/hudson/matrix/MatrixProject.java

```diff
@@ -727,7 +732,14 @@ public MatrixConfiguration getItem(Combination c) {
         return configurations.get(c);
     }
 
-    public File getRootDirFor(MatrixConfiguration child) {
+    /**
+     * Gets a root directory for the combination.
+     * Creates the whole directory hierarchy on-demand.
+     * @param combination
```

Comment:
typo: configuration. As long I am concerned we can omit this entirely, it does not add much value.